test/__init__.py

```diff
@@ -80,23 +81,10 @@ def assert_md5_equal_to_line_in_file(testCase, filename, checksum_file, msg=None
 
     testCase.assertEqual(hash_md5.hexdigest(), expected_checksum, msg=msg)
 
+@pytest.mark.usefixtures('tmpdir_class')
 class TestCaseWithTmp(unittest.TestCase):
     'Base class for tests that use tempDir'
 
-    @classmethod
```

Comment:
This removal fixes the issue I was having, but does it break `unittest`?

Reply:
@tomkinsc What mode of breakage are you thinking about?   Can you maybe add a test to test for it?   Current test suite doesn't seem to break.

Reply:
Incompatibility when testing under the built-in `unittest` module, which I suppose we don't need to worry about if we're going all-in on pytest.

Reply:
@tomkinsc I've been adding tests to the test suite that are pytest-based only, which would not be run if pytest is not used.  Is there a goal of keeping the test suite runnable without pytest?

Reply:
I don't think we've talked about it(?). @dpark01 @yesimon @haydenm, what are your thoughts?

Reply:
I'm for moving to pytest

Reply:
Originally I think we had it all set up to be backwards compatible with `unittest`. But I think we've already crossed the bridge to being pytest-dependent. A fair amount of the metagenomics tests are not `unittest` compatible. Anyway I wouldn't worry about it at this point...